core/qthelper.cpp

```diff
@@ -402,8 +402,12 @@ static int parseExif(const QString &filename, easyexif::EXIFInfo &exif)
 extern "C" timestamp_t picture_get_timestamp(const char *filename)
 {
 	easyexif::EXIFInfo exif;
-	if (parseExif(localFilePath(QString(filename)), exif) != PARSE_EXIF_SUCCESS)
-		return 0;
+	if (parseExif(localFilePath(QString(filename)), exif) != PARSE_EXIF_SUCCESS) {
+		// If we couldn't parse EXIF data, use file creation date.
+		// TODO: QFileInfo::created is deprecated in newer Qt versions.
+		QDateTime created = QFileInfo(QString(filename)).created();
```

Comment:
I cannot really think of a situation where this would be the correct thing (i.e. the time an underwater picture or picture at a dive site were taken). This will run on a desktop computer and the time the file is created there is most likely a different one (namely the time of the transfer from camera/phone/whatever). I would rather leave it at 0.

Reply:
I was hoping that the file-creation date is preserved on copying. This seems to be confirmed in by a quick look through my `Pictures` directory. Perhaps a Linux thing?

Reply:
> I was hoping that the file-creation date is preserved on copying. This seems to be confirmed in by a quick look through my Pictures directory. Perhaps a Linux thing?

Hmmm, difficult story. I also checked my pictures and videos under Windows 10. Videos which I didn't modify (no time to cut my action cam videos) do have correct file created date.
Same for pictures which I didn't modify. But as soon as I modify a picture with a picture editor the file "date created" and "date modified" is changed.
But I'm still with Berthold. Why not do:
- 1st prio: EXIF
- 2nd prio: EXIF equivalent for video (future feature)
- 3nd prio: file date created

Why should a incorrect file data created be worse compared to having no date/time info at all? We do have the "timestamp doesn't match dive date/time" dialog for covering this.
Maybe the dialog should also show the source of the timestamp (EXIF or file date).